### PR TITLE
feat: Grant pull subscription permissions for external service account

### PR DESCRIPTION
**Pull request by askoriy**
It is useful to grant permissions for the service account that will subscribe to that pull_subscription just after creating it.
Especially if granting access to the service account from another project.

Also as Kafka-connect needs additional `roles/pubsub.viewer` role for subscribing to the topic, `service_account_viewer_role` switch is provided.

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by morgante** [comment]
I'm not sure we want to include this in the module directly.

**Comment by askoriy** [comment]
Yes, it's really not useful for granting permissions inside one project or administration domain.
However, this is the most obvious way to grant permissions for external subscribers because both subscription and IAM configured and managed in one place. At least we need it.

File: README.md [hunk 1]
```diff
@@ -38,14 +38,16 @@ module "pubsub" {
...
+      name                        = "pull"                                               // required
+      ack_deadline_seconds        = 20                                                   // optional
+      dead_letter_topic           = "projects/my-pubsub-project/topics/example-dl-topic" // optional
+      max_delivery_attempts       = 5                                                    // optional
+      maximum_backoff             = "600s"                                               // optional
+      minimum_backoff             = "300s"                                               // optional
+      filter                      = "attributes.domain = \"com\""                        // optional
+      enable_message_ordering     = true                                                 // optional
+      service_account             = "service2@project2.iam.gserviceaccount.com"          // optional
+      service_account_viewer_role = true                                                 // optional
```
Comment (morgante) [inline README.md:50]: Could we simply always set this? I'm not sure we need the flexibility.

**Review by morgante — APPROVED** [review]

**Pull request merged**